src/Rector/Deprecation/PathAliasManagerServiceNameRector.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation;
+
+use PhpParser\Node;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * Renames deprecated Drupal::services('path.alias_manager') argument.
+ *
+ * See https://www.drupal.org/node/3092086 for change record.
+ *
+ * What is covered:
+ *  - Replacement in Drupal::service() calls.
+ */
+final class PathAliasManagerServiceNameRector extends DrupalServiceRenameBase {
```

Comment:
For now (we may revisit this), we've tried to keep one rector class per deprecation. This seems the simplest to jump into even if it's a bit verbose in cases. We may revisit this later.